### PR TITLE
fixing todo's and small lint fixes

### PR DESCRIPTION
**Pull request by sendilkumarn**
- Please make sure the below checklist is followed for Pull Requests.

- [x] [Travis tests](https://travis-ci.org/jhipster/generator-jhipster/pull_requests) are green
- [x] Tests are added where necessary
- [x] Coding Rules & Commit Guidelines as per our [CONTRIBUTING.md document](https://github.com/jhipster/generator-jhipster/blob/master/CONTRIBUTING.md) are followed

### CODE REVIEW

File: generators/client/templates/angular/src/main/webapp/app/layouts/error/_error.component.ts [hunk 1]
```diff
@@ -28,12 +29,20 @@ export class ErrorComponent implements OnInit {
...
     }
 
     ngOnInit() {
-        <%_ // TODO need to see how the error message can be passed here _%>
+        this.route.data.subscribe((routeData) => {
+            if (routeData.error403) {
+                this.error403 = routeData.error403;
+            }
+            if (routeData.errorMessage) {
+                this.errorMessage = routeData.errorMessage;
```
Comment (sendilkumarn) [inline generators/client/templates/angular/src/main/webapp/app/layouts/error/_error.component.ts]: interesting that we haven't called `error page` from anywhere ? Are we ?

File: generators/client/templates/angular/src/main/webapp/content/css/_global.css [hunk 2]
```diff
@@ -82,13 +82,6 @@ Generic styles
     opacity: 1;
 }
 
-/*FIXME this is to support grids in table class; */
-table td[class*=col-], table th[class*=col-] {
```
Comment (sendilkumarn) [inline generators/client/templates/angular/src/main/webapp/content/css/_global.css]: None of the table's td / th has this class.

Comment (deepu105) [inline generators/client/templates/angular/src/main/webapp/content/css/_global.css]: these were for the entity. did you check that?

File: generators/client/templates/angular/src/main/webapp/app/blocks/interceptor/_auth-expired.interceptor.ts [hunk 3]
```diff
@@ -43,7 +43,6 @@ export class AuthExpiredInterceptor extends HttpInterceptor {
 <%_ } if (authenticationType === 'session') { _%>
     constructor(private injector: Injector,
         private stateStorageService: StateStorageService) {
-
```
Comment (deepu105) [inline generators/client/templates/angular/src/main/webapp/app/blocks/interceptor/_auth-expired.interceptor.ts]: if this was lint fail how come the tests were passing it should have failed as I made lint not to force run

File: generators/client/templates/angular/src/main/webapp/content/css/_global.css (hunk 2 above)
Comment (sendilkumarn) [inline generators/client/templates/angular/src/main/webapp/content/css/_global.css]: Nope

Comment (sendilkumarn) [inline generators/client/templates/angular/src/main/webapp/content/css/_global.css]: Nope

**Pull request merged**